fractal_server/app/routes/auth.py

```diff
@@ -43,8 +44,16 @@
 # Include users routes, after removing DELETE endpoint (ref
 # https://github.com/fastapi-users/fastapi-users/discussions/606)
 users_router = fastapi_users.get_users_router(UserRead, UserUpdate)
```

Comment:
Let's clarify the comment, to mention that we are removing some `/me` endpoints to replace them with our own endpoints below.